insights/client/config.py

```diff
@@ -681,6 +682,52 @@ def _validate_options(self):
         '''
         Make sure there are no conflicting or invalid options
         '''
+
+        def _validate_obfuscation_options():
+            # When old switches are set explicitly, even set as False
+            if self.obfuscate is not None or self.obfuscate_hostname is not None:
+                # Warning deprecation only on RHEL 8+ and from egg v 3.6.0 (planned)
+                if self._print_errors and get_rhel_version() > 7:
+                    egg_ver = get_version_info()['core_version'].split('-')[0]
+                    if [int(i) for i in egg_ver.split('.')] >= [3, 6, 0]:
```

Comment:
Can you maybe export this code into some function (e.g. create a function in `utilities.py`) which would return a tuple? For example something like this:
```python
def get_egg_version():
    core_version = get_version_info()['core_version'].split('-')[0]
    return tuple(int(i) for i in min_version.split('.'))
````

Reply:
It's totally okay to me to wrap it with a function. As thinking here were the only usage point, didn't do that. Will do it. Thanks.